### ISSUE TITLE
✔️ 🐞 Bug Report: CONTAINER not work properly

### ISSUE THREAD

**Issue by drau1** [elementor/elementor#17064]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest development version of Elementor.

## Description

I create different test, using cantainer inna container, change orientation an all possibilities.
At the end what i have in backend preview is not the same of real preview.
My image insiede containere are vertically aligned and not like me put in backend
## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro) with development versions.
- [x] This bug happens with a vanilla WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
[system-info-www.drau.it-29-11-2021.txt](https://github.com/elementor/elementor/files/7617277/system-info-www.drau.it-29-11-2021.txt)

</details
![2021-11-29-11-00-www drau it](https://user-images.githubusercontent.com/94255240/143847366-0e81d795-e297-46eb-8855-a3141b46b2db.png)
![Uploading 2021-11-29-11-01-www.drau.it.png…]()
>

**Comment by shilo-ey** [comment]
Hey @drau1 

Thanks, can you share a screen recording, image or a template export?

Thanks

**Comment by JKHopf** [comment]
I think I might have a similar issue. Vertical alignment of a column is not working (e.g. will not do anything other than top align) if a container is nested inside another container (i.e. inner section inside a section). Vertical alignment does work when the container is not nested. 
I have updated to most recent elementor pro.

Example picture: Layout is for the right column with text in it. 

![image](https://user-images.githubusercontent.com/36061539/152668839-bb403c61-b918-4bc9-a3e1-8c7b77cdd3e8.png)

**Comment by nicholaszein** [comment]
Hello there @drau1 and @JKHopf!

I think by now you already know this, but it doesn't hurt to say:

### 📢 We're happy to announce that the issue you raised was resolved in Elementor **v3.8**! 🥳 

The Flexbox Container experiment is now stable and it does not present the reported behavior.

✔️ Please feel free to check it out and update your plugin to the new version!

Cheers 🥂
